modules/ossm-threescale-integrate.adoc

```diff
@@ -17,7 +17,7 @@ You can use these examples to configure requests to your services using the 3sca
 * {ProductName} prerequisites
 * Ensure Mixer policy enforcement is enabled. Update Mixer policy enforcement section provides instructions to check the current Mixer policy enforcement status and enable policy enforcement.
 * Mixer policy and telemetry must be enabled if you are using a mixer plug-in.
-** You will need to properly configure the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/service_mesh/service-mesh-installation#ossm-control-plane-deploy-2x_installing-ossm-v2x[Service Mesh Control Plane (SMCP)] when upgrading.
+** You will need to properly configure the link:https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/installing-ossm.html[Service Mesh Control Plane (SMCP)] when upgrading.
```

Comment:
@dfennessy I don't think you need a link here at all. Also, because this is a module, the published doc doesn't support linking within the content.

But looking at the content, I think that we can just remove the link.